### ISSUE TITLE
Add LXC extra configuration line by line with PV_LXC_EXTRA_CONF

### ISSUE THREAD

**Issue by anpopa** [pantacor/pvr#7]
Currently adding extra configuration in PV_LXC_EXTRA_CONF result in sourcing the value content directly without any option to define multiple lines in the configuration file.
For example an array of strings like:

`
"PV_LXC_EXTRA_CONF" : [
    "lxc.mount.entry = <mount1>\n",
    "lxc.mount.entry = <mount2>\n"
],
`
will generate the invalid lxc.container.conf output:

`
[
lxc.mount.entry = <mount1>
lxc.mount.entry = <mount1>
]
`
The issue is how the value is sourced in the template file: https://github.com/pantacor/pvr/blob/master/templates/builtin-lxc-docker.go#L156
Is very hard to work with a very long line in configuration files. Is better to split the extra conf with an array of strings. Other options like PV_LXC_CAP_KEEP are actually handling the elements as key:value which makes the split possible.

**Comment by asac** [comment]
Yes, I agree that this is a worthwhile improvement to make with pvr. Would have to try how to make a good improvement without breaking the current format of that CONF entry....

**Comment by asac** [comment]
if you feel you have a PR I would happily take it.... please use our gitlab.com/pantacor/pvr code base to submit a MR though as that is the current primary source tree...
